src/makeInfinityAdder.js

```diff
@@ -4,9 +4,18 @@
  *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  const adder = function(a) {
+    if (a === undefined) {
+      return 0;
```

Comment:
don't forget to save previous sum because it shows 0 in case:
```
adder(5)(5)(5)
adder(4)
adder()
```
but it should be 19